pdks/Sky130/scripts/export.py

```diff
@@ -2,9 +2,17 @@
 #
 # Exports PDK data in Vlsir schema format (protobufs).
 
-# This requires that Vlsir schema bindings and VlsirTools both be importable by
-# Python. For development you can invoke with:
-#   PYTHONPATH=../VlsirTools/:../bindings/python/ ./proto2spice.py
+# This script requires that the following Vlsir ecosystem packages be available
```

Comment:
I think it's fair that a script inside the `scripts/` directory of a python package-repo needs... the same dependencies as the package, right? 

In other words if one runs `pip install sky130_hdl21`, would it exclude anything required by this script?

Reply:
I think you are saying that this comment is redundant and that I should remove it. I have reworded it to be less obvious and more the hint for developers I inteded it as (especially those not fully cognisant of the dependency tree for hdl21, like me). I will not argue to keep it